### PR TITLE
EDSC-1986: Implements panel based layouts

### PR DESCRIPTION
**Pull request by abbottry**

### CODE REVIEW

File: app/assets/javascripts/modules/map/map.js.coffee [hunk 1]
```diff
@@ -72,29 +72,32 @@ ns.Map = do (window,
...
 
-      if !@isMinimap
-        @time = ko.computed(@_computeTime, this)
-        map.fire('edsc.visiblecollectionschange', collections: page.project.visibleCollections())
-        @_granuleVisualizationSubscription = page.project.visibleCollections.subscribe (collections) ->
-          map.fire('edsc.visiblecollectionschange', collections: collections)
+      @legendControl = new LegendControl(position: 'topleft')
+      map.addControl(@legendControl)
+
+      # if !@isMinimap
```
Comment (abbottry) [inline app/assets/javascripts/modules/map/map.js.coffee]: Let's kill this

File: app/assets/javascripts/modules/map/map.js.coffee [hunk 2]
```diff
@@ -72,29 +72,32 @@ ns.Map = do (window,
...
+        map.fire('edsc.visiblecollectionschange', collections: collections)
       if @isMinimap
         map.addControl(new SpatialSelection(@isMinimap))
-        map.dragging.disable()
-        map.touchZoom.disable()
-        map.doubleClickZoom.disable()
-        map.scrollWheelZoom.disable()
-        # Disable tap handler, if present.
-        map.tap.disable() if (map.tap)
+        # map.dragging.disable()
```
Comment (abbottry) [inline app/assets/javascripts/modules/map/map.js.coffee]: Can kill all of this too.

File: app/assets/javascripts/projects.js [hunk 3]
```diff
@@ -29,3 +29,116 @@
 //= require bootstrap/bootstrap
 //= require jquery.echoforms-full.min
 //= require modules/echoformsext.coffee.erb
+//= require components
```
Comment (abbottry) [inline app/assets/javascripts/projects.js]: This seems.. super outdated, is it actually relevant? jQuery 1.3 😬

File: app/assets/javascripts/search.js [hunk 4]
```diff
@@ -28,6 +28,5 @@
 
 $(function(){
```
Comment (abbottry) [inline app/assets/javascripts/search.js]: Does this serve a purpose? Removing the `console.warn` is valid but does it make sense to leave the rest of it?

File: app/assets/stylesheets/modules/projects.css.scss [hunk 5]
```diff
@@ -320,63 +172,61 @@ $card-background: rgba(235,235,235,1);
     padding-right: 25px;
   }
 
-  .modal-body {
-    ul > li {
-      padding: 10px 15px;
-    }
+  // ul > li {
```
Comment (abbottry) [inline app/assets/stylesheets/modules/projects.css.scss]: 👋 buh bye!

Comment (mreese84) [reply app/assets/stylesheets/modules/projects.css.scss]: Best PR comment I've seen in a while.

File: app/views/shared/components/project_list/_project_list.html.erb [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+              </div>
+              <div class="project-list-item-loading-row">
+                <div class="loading-content project-list-item-loading-capabilities"><span class="loading-content-inner"></span></div>
+                <div class="loading-content project-list-item-loading-options"><span class="loading-content-inner"></span></div>
+              </div>
+            </div>
+            <!-- /ko -->
+            <!-- ko if: isLoadingComplete() -->
+              <span class="project-list-item-stat project-list-item-stat-granules">
+                <span data-bind="text: collection.granule_hits, fadeVisible: isLoadingComplete"></span> <!--ko text: collection.granule_hits != 1 ? 'Granules' : 'Granule' --><!--/ko --></span>
```
Comment (macrouch) [inline app/views/shared/components/project_list/_project_list.html.erb]: I think there is an extra `</span>` here

**Comment by trevorlang** [comment]
Looks good to me 👍

**Pull request merged**